formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -235,16 +235,16 @@ class myClassWithAnnotatedReturnType3
 class myClassWithAnnotatedReturnType3_annotated_constructor:
   int =>
   new {
-    method x: int;
-    method y: int
+    pub x: int;
```

Comment:
Do you think we should eventuallyunify these `class_instance_type` with object instance types too? They allow much more to be specified than object types (for example public vs. private, and even values). But could it be that class instance types could be a syntactic superset of object instance types?